city_scrapers/spiders/clovis_city_council.py

```diff
@@ -0,0 +1,106 @@
+import re
+import string
+from datetime import date, datetime
+from time import time
+
+from city_scrapers_core.constants import CITY_COUNCIL
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ClovisCityCouncilSpider(CityScrapersSpider):
+    name = "clovis_city_council"
+    agency = "Clovis City Council"
+    timezone = "America/Chicago"
+
+    #start_urls = ["https://cityofclovis.com/government/city-council/city-council-agendas/"]
+    start_urls = ["https://meetings.municode.com/PublishPage?cid=CLOVIS&ppid=5157d66d-a361-43e8-87a4-3d5eca4821de&p=1"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        for item in response.css(".div-table tr"):
+            title = item.css("td:nth-child(1)::text").get()
+            if title:
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(item),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        title = item.css("td:nth-child(1)::text").get()
+        return title.strip()
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return CITY_COUNCIL
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        #start time
+        startTime = item.css("td:nth-child(3)::text").get()
+        time = re.search('(.+?) ', startTime).group(1)
+
+        #start date
+        startDate = item.css("td:nth-child(2)::text").get()
+        date = "8/1/2022"
+
+        #trying to extract date
+        #date = re.search('([0-9]/[0-9]/[0-9]{4})', startDate.strip()).group(0)
```

Comment:
[0-9] can be written as \d (for digit)

you also want month & day to be \d{1,2} (1-2 digits)

Reply:
@jpt-c got it, thanks. 

OK do i dare open a can of worms and inquire about military time. 
start time returned as string '6:00 PM'
datetime interprets this as 6AM since it doesn't read the PM.

Reply:
AM & PM can be parsed with strptime w/ the %p specifier

You can also use dateutil to parse without a format string:
```
import dateutil.parser

dateutil.parser.parse(time_string)
```

Reply:
@jpt-c 

i want you to know that i spent an embarrassing amount of time realizing it's parser().parse(dt_obj), not parser.parse(dt_obj). this spider is now d-o-n-e.

thank you for always helping me. i promise these are going to get faster and i am working on these everyday. my workflow is i figure out the scrapy commands for each data item, and then when i have those commands figured out, i insert them into the spider.